content/docs/ops/runbook/restoring-rds.md

```diff
@@ -0,0 +1,89 @@
+---
+menu:
+  docs:
+    parent: runbook
+layout: ops
+
+title: Restoring RDS
+---
+
+## Introduction
+Many of the major services for the cloud.gov platform are backed by Postgres databases provided by RDS.  In the case of outages or corruption, it is imperative an operator be able to quickly find the affected database and restore it from a snapshot if possible.  These instructions augment the official RDS documentation.
+
+## Find database identifier
+
+### CloudFoundry Applications
+Coordinate with the tenant, informing them that a restore is going to take place:
+1. Shared database plans do not include backups and restores
+2. Dedicated RDS plans include 1 day of backups
+3. The restore will result in data lost after the point of the most recent snapshot
+4. This restore needs to be initiated before the day passes, otherwise new snapshots will be written
+
+Once the tenant consents, identify their Organization and Space.  Then perform the following:
+```sh
+cf target -s SPACE -o ORGANIZATION
+```
+
+Print environment variables for the affected application:
+```sh
+cf env APP
+```
+
+The resulting JSON contains the "host" name within the credentials map.  This will be the database identifier needed below for restoration.
+
+### Platform Applications
+Databases created by Terraform store their credentials in a Terraform State Bucket.  This can be found in S3:
+```sh
+aws s3 cp s3://${TF_STATE_BUCKET}/cf-${ENVIRONMENT}/terraform.tfstate tmp/state.file
+```
+
+To retrieve the RDS instance provisioned for `staging concourse`:
+```sh
+cat tmp/sample.file | grep staging_concourse_rds_host | less
+```
+
+The value displayed is the database to restore.  To restore the BOSH instance grep for `bosh_rds_host_curr`.
+
+NOTE: These values can also be found in the AWS RDS console.  This can be faster.
+
+## Restore the database
+Refer to the [RDS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_RestoreFromSnapshot.html) for restoring a database.
+
+For the step that specifies choosing the database to restore, this database will be the respective database we found in the steps above.
+
+Take note to carefully set options matching the configuration of the target database.  Specifically pay attention to these options:
+
+1) Database publicly accessible from the internet (yes/no)
+2) Instance size (m4_large, etc.)
+3) Multi-zone (yes/no)
```

Comment:
I really think that VPC is super important to call out here too.

Reply:
It is and it's noted in the following paragraph.  Remember we couldn't set it until after restoring the snapshot?

Reply:
VPC was set, but the security groups could not be set upon restore.

Reply:
You're right!  added.